control/fixtures/fixtures.go

```diff
@@ -155,3 +156,20 @@ func (m MockRequestedMetric) Version() int {
 func (m MockRequestedMetric) Namespace() core.Namespace {
 	return m.namespace
 }
+
+func NewExecutablePlugin(a plugin.Arg, path string) (*plugin.ExecutablePlugin, error) {
+	// Travis optimization: Try starting the plugin three times before finally
+	// returning an error
```

Comment:
Is it problem with Travis or framework? Is it workaround that should be addressed in the future?